### PR TITLE
Add ability to create DASH videos through the Java client.

### PR DESCRIPTION
**Pull request by mdeimel**
This also covers issues #19 and #20

### CODE REVIEW

File: src/main/java/com/brightcove/zencoder/client/model/DeliveryFormat.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ * SOFTWARE OR ITS DERIVATIVES.
+ *
+ * "Brightcove" is a registered trademark of Brightcove Inc.
+ */
+package com.brightcove.zencoder.client.model;
+
+/**
+ * @see https://app.zencoder.com/docs/api/encoding/segmented-streaming/streaming-delivery-format
+ */
+public enum DeliveryFormat {
```
Comment (mdeimel) [inline src/main/java/com/brightcove/zencoder/client/model/DeliveryFormat.java]: This seemed like the appropriate name for the class, following the standards of other names, and the JSON reference of `streaming-delivery-format`.

File: src/main/java/com/brightcove/zencoder/client/request/ZencoderOutput.java [hunk 2]
```diff
@@ -977,12 +987,17 @@ public void setHlsOptimizedTs(Boolean hls_optimized_ts) {
         this.hls_optimized_ts = hls_optimized_ts;
     }
 
-    public SegmentingType getPrepareForSegmenting() {
+    public List<SegmentingType> getPrepareForSegmenting() {
         return prepare_for_segmenting;
     }
 
     public void setPrepareForSegmenting(SegmentingType prepare_for_segmenting) {
```
Comment (mdeimel) [inline src/main/java/com/brightcove/zencoder/client/request/ZencoderOutput.java]: Now that `prepare_for_segmenting` is a `List`, a `set` method would not seem necessary, but in order to maintain backward compatibility, leaving this in there, and using it to reset the `List` to a single entry.

**Comment by GeneticGenesis** [comment]
Hi @mdeimel,

Sorry we didn't spot this, someone will take a look ASAP.

Cheers,

**Comment by mdeimel** [comment]
Great - thanks @GeneticGenesis and @skidder for the merge!

**Comment by GeneticGenesis** [comment]
No problem. Sorry again for the delay.

Cheers,

**Pull request merged**